english/Rules to Success/Rules.txt

```diff
@@ -0,0 +1,17 @@
+Get comfortable being uncomfortable. Get outside your comfort zone on a regular basis. The more you do, the wider that area becomes. Things that used to seem impossible can become part of your everyday life.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.